ydb/aio/query/session.py

```diff
@@ -115,6 +115,7 @@ async def execute(
         parameters: dict = None,
         syntax: base.QuerySyntax = None,
         exec_mode: base.QueryExecMode = None,
+        stats_mode: Optional[base.QueryStatsMode] = None,
```

Comment:
move the parameter to end of make it named only for prevent break compatibility (here and all other places).